pyexcelerate/Style.py

```diff
@@ -84,6 +112,8 @@ def get_xml_string(self):
             tag.append('applyFill="1" fillId="%d"' % (self._fill.id + 1))
         if not self._borders is None:
             tag.append('applyBorder="1" borderId="%d"' % (self._borders.id))
+        if not self._quote_prefix is None:
+            tag.append('quotePrefix="1"')
```

Comment:
I think this line has to be set based on the truthiness of quote prefix? The condition will write 1 even if it's set to false for example.